SEP-0002-revised.md

```diff
@@ -0,0 +1,245 @@
+Board meetings shall provide at least one week (7) days notice.
+Notice of meetings shall specify the place, day, and hour of meeting.
+The purpose of the meeting need not be specified.
+
+* **Board officers**: The officers of the board shall be a board chair, vice-chair, and secretary, all of whom shall be chosen by, and serve at the pleasure of, the board.
+Officers shall serve one year terms and can serve consecutive terms.
+
+* **Board Chair**: The board chair shall lead the board in performing its duties and responsibilities, including, if present, leading all meetings of the board, and shall perform all other duties incident to the office or properly required by the board.
+
+* **Vice-chair**: In the absence of the board chair, the vice-chair shall perform the duties of the board chair.
+When so acting, the vice-chair shall have all the powers of and be subject to all the restrictions upon the board chair.
+
+* **Secretary**: The secretary shall be responsible for keeping the minutes of all meetings and actions of the board.
+The minutes of each meeting shall state the time and place that it was held, the members present, and such other information as shall be necessary to determine the actions taken and whether the meeting was held in accordance with the law and these bylaws.
+
+* **Advise and Consent**: The steering committee shall present the state of the project and future plans at meetings of the board.
+The advisory board shall advise and consent on those plans.
+
+## The Steering Committee
+The role of the steering committee which is headed by the Lead Developer, is to manage the day to day operations of the SunPy organization.
+The steering committee shall be responsible for the running of the project under advisement of the board.
+This includes:
+
+* holding regular meetings with the developer community (at least monthly)
+* making regular reports to the board (at least quarterly)
+* having ownership privileges to the SunPy GitHub organization
+* creating, maintaining and enforcing the SunPy code of conduct (see below)
+* accepting or rejecting SEPs based on community consensus
+* delegating responsibility and authority to other community roles
+* planing and manage finances and financial processes of the project.
+
+### Composition
+
+The steering committee is formed of three roles:
+
+* **The lead developer** chairs the steering committee and is responsible for holding regular (at least monthly) meetings, and ensuring that the committee is represented at all board meetings.
+* **The deputy lead developer** in the absence of the lead developer, the deputy shall perform all the duties of the lead developer.
+* **The financial officer** the financial officer is primarily responsible for ensuring the health of the project finances and setting up processes for managing expenses and reporting to the board on financial matters.
+
```

Comment:
I don't know if there is any precedence for this, but it would be nice to have some other roles like Senior Developer or something like that for longtime, active contributors of the project that currently have no formal title, e.g. @nabobalis.

Reply:
Those other kinds of roles, i.e. the ones listed here: https://sunpy.org/project/#community-roles have not been detailed in our governance, but created by the lead developer / community. I think that your suggestion would fall under that?

Reply:
Seems like the emeritus status below may be similar to that role though maybe the role should be better defined but right now it just seems like people who have stopped contributing.

Reply:
Yeah, it's for people who were active and who no longer are.